src/styles/utils/_functions.scss

```diff
@@ -0,0 +1,27 @@
+@for $monthLength from 28 through 31 {
+  .calendar__container--month-length-#{$monthLength}
+  .calendar__day:not(:nth-child(-n+#{$monthLength})) {
+    display: none;
+  }
+}
```

Comment:
Loops is not a functions so move it to its blocks

Reply:
Done